### ISSUE TITLE
abnormal peripapillary microvascular network

### ISSUE THREAD

**Issue by Sukramg** [obophenotype/human-phenotype-ontology#7707]
**Preferred term label:**
Abnormal Peripapillary Microvascular Network

**Synonyms**
peripapillary microangiopathy

**Definition (free text, please give PubMed ID)**
A reduction of the peripapillary microvascular network
doi: 10.1111/aos.13817

**Parent term (use hpo.jax.org/app)**
optic nerve disorder MONDO:0002135

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Leber hereditary optic neuropathy, other chronic optic
neuropathies

**Your nano-attribution (ORCID)**
0000-0002-6601-4486

**Comment by pnrobinson** [comment]
@Sukramg 
would this be good?

label:
def: The density of the deep peripapillary microvascular network (i.e., the number of blood vessels) is below the lower limit of normal. This finding is observable by optical coherence tomography angiography.
Comment: See figure 1 of PMID:29073242

**Comment by Sukramg** [comment]
Your suggestion is more specific in terms of the diagnostic method (optical coherence tomography angiography) and what exactly is meant by the abnormality (density of the microvasculature). 
Ultimately, it is more precise and therefore more suitable.

I think it makes sense to create a specific term such as "abnormal peripapillary morphology". This would enable more precise diagnosis and research in this specific area. "Abnormal optic disc morphology" is a helpful parent term, perhaps downstream?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000771
- label: Abnormal peripapillary microvascular network
- parents: [Abnormal optic disc morphology (HP:0012795)](https://hpo.jax.org/app/browse/term/HP:0012795)
- pmid: 
- def: The density of the deep peripapillary microvascular network (i.e., the number of blood vessels) is below the lower limit of normal. This finding is observable by optical coherence tomography angiography.
- comment: 
- orcid: orcid:0000-0002-6601-4486
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7707
- synonyms:
